tests/data/config.yaml

```diff
@@ -35,7 +35,7 @@ solver:
     sound_speed:
       path: ./tests/data/2022/NCL1/ctd/CTD_NCL1_Ch_Mi
     travel_times:
-      path: ./tests/data/2022/NCL1/**/WG_*/pxp_tt
+      path: ./tests/data/2022/NCL1/**/WG_*/pxp_tt*
```

Comment:
Let's not change this to also pull `j2k`. A user would either have to specify the regular `pxp_tt` file OR `pxp_tt_j2k`, not both.